Source/WebCore/rendering/CSSFilter.cpp

```diff
@@ -60,9 +60,18 @@ RefPtr<CSSFilter> CSSFilter::create(RenderElement& renderer, const FilterOperati
     return filter;
 }
 
-RefPtr<CSSFilter> CSSFilter::create(Vector<Ref<FilterFunction>>&& functions)
+Ref<CSSFilter> CSSFilter::create(Vector<Ref<FilterFunction>>&& functions)
 {
-    return adoptRef(new CSSFilter(WTFMove(functions)));
+    return adoptRef(*new CSSFilter(WTFMove(functions)));
+}
+
+Ref<CSSFilter> CSSFilter::create(Vector<Ref<FilterFunction>>&& functions, OptionSet<FilterRenderingMode> filterRenderingModes, const FloatSize& filterScale, const FloatRect& filterRegion)
+{
+    Ref filter = adoptRef(*new CSSFilter(WTFMove(functions)));
+    filter->setFilterRenderingModes(filterRenderingModes);
```

Comment:
would be nice if these were in a constructor.

Reply:
Sure, I'll fix tomorrow.